### PR TITLE
SPM-755 Add bulk remediation for systems and advisories

### PR DESCRIPTION
**Pull request by jiridostal**
This PR implement the Patch side of bulk remediation.
**There is a bug that prevents the remediation from creating - the wizard goes to the last step and than fails, platform is working on it**

The purpose is to demonstrate the behavior on Patch side.

### CODE REVIEW

**Review by mkholjuraev — CHANGES_REQUESTED** [review]
I am afraid the PatchRemediationButton component is missing from the PR commit.

**Comment by jiridostal** [comment]
Fixed!

File: src/Utilities/Helpers.js [hunk 1]
```diff
@@ -154,6 +168,20 @@ export const remediationProvider = (issues, systems, remediationIdentifier) => {
         : false;
 };
 
+export async function remediationProviderWithPairs(issues, createPairs, transformFunc) {
+    if (issues) {
+        const pairsCreated = await createPairs(issues);
+        const res = transformFunc(pairsCreated);
+        console.log(await res);
```
Comment (mkholjuraev) [inline src/Utilities/Helpers.js:175]: I believe this needs to be deleted :)

**Review by mkholjuraev — COMMENTED** [review]
It looks that 2 remediation modals are being mounted in the Systems page, when I close one model, the other is displayed. Otherwise working as described.

**Pull request merged**